libcontainer/mount_linux.go

```diff
@@ -47,19 +50,31 @@ func (e *mountError) Unwrap() error {
 	return e.err
 }
 
-// mount is a simple unix.Mount wrapper. If procfd is not empty, it is used
-// instead of target (and the target is only used to add context to an error).
-func mount(source, target, procfd, fstype string, flags uintptr, data string) error {
+// mount is a simple unix.Mount wrapper, returning an error with more context
+// in case it failed.
+func mount(source, target, fstype string, flags uintptr, data string) error {
+	return mountViaFDs(source, "", target, "", fstype, flags, data)
+}
+
+// mountViaFDs is a unix.Mount wrapper which uses srcFD instead of source, and
+// dstFD instead of target, unless those are empty. In such cases, source and
```

Comment:
IMHO, it seems worth documenting that srcFD and dstFD are a string and expected to have the format `/proc/self/fd/<int>` or something like that.

That is not obvious for a user that an fd is of type string and should be in that format, I think. It can be deduced by looking at how the code is used and it is arguable if before it was clearer because the var name was "procfd". I do think, though, that a small documentation sentence can help :)

What do you think?

Reply:
It's clear from the function proto that FDs are strings, but it makes sense to point out they are like "/proc/self/fd/N".

Added.